src/modules/networking/tests/NetworkingTests.cpp

```diff
@@ -244,29 +256,44 @@ namespace OSConfig::Platform::Tests
         "GENERAL.STATE:                          100 (connected)\n"
         "GENERAL.CONNECTION:                     Wired connection 1\n";
 
+        std::vector<std::string> returnValuesDataMissing = {g_testCommandOutputNames, testCommandOutputInterfaceTypesNmcliDataMissing, "", g_testIpData, g_testCommandOutputDefaultGateways, g_testCommandOutputDnsServers};
+
         MMI_JSON_STRING payload;
         int payloadSizeBytes;
         NetworkingObjectTest testModule(g_maxPayloadSizeBytes);
-        testModule.returnValues = g_returnValues;
-        testModule.returnValues[1] = testCommandOutputInterfaceTypesNmcliNoData;
+        testModule.returnValues = returnValuesDataMissing;
         int result = testModule.Get(nullptr, nullptr, nullptr, &payload, &payloadSizeBytes);
 
         EXPECT_EQ(result, MMI_OK);
 
         std::string resultString(payload, payloadSizeBytes);
-        EXPECT_STREQ(resultString.c_str(), payloadNmcliInterfaceTypesDataMissing);
+        EXPECT_STREQ(resultString.c_str(), payloadInterfaceTypesDataMissing);
 
         EXPECT_NE(payload, nullptr);
         delete payload;
 
+        std::vector<std::string> returnValuesNetworkManagerEnabled = {g_testCommandOutputNames, g_testCommandOutputInterfaceTypesNmcli, g_testIpData, g_testCommandOutputDefaultGateways, g_testCommandOutputDnsServers};
         testModule.runCommandCount = 0;
-        testModule.returnValues = { g_testCommandOutputNames, "", g_testCommandOutputInterfaceTypesNetworkctl, g_testIpData, g_testCommandOutputDefaultGateways, g_testCommandOutputDnsServers };
+        testModule.returnValues = returnValuesNetworkManagerEnabled;
         result = testModule.Get(nullptr, nullptr, nullptr, &payload, &payloadSizeBytes);
```

Comment:
>testModule.Get(nullptr [](http://example.com/codeflow?start=17&length=22)

MmiGet with a null session handle must fail. So the vase class Get method must.

Reply:
NetworkingObjectBase::Get does not use any of the arguments passed here as nullptr. It may be best to simply remove the arguments from the method prototype (as is done for the Tpm::Get method)?